### ISSUE TITLE
validation error in helm chart templates license 

### ISSUE THREAD

**Issue by isandeep41** [apache/druid#11838]
#### Issue
Helm install throw validation error due to incorrect comment format
```sh
$ helm version
version.BuildInfo{Version:"v3.7.0", GitCommit:"eeac83883cb4014fe60267ec6373570374ce770b", GitTreeState:"clean", GoVersion:"go1.17"}

$ helm install druid . --values values.yaml --namespace default 
Error: INSTALLATION FAILED: unable to build kubernetes objects from release manifest: error validating "": error validating data: apiVersion not set
```

#### Solution

To use: [Helm Template Comments](https://helm.sh/docs/chart_best_practices/templates/#comments-yaml-comments-vs-template-comments)
```yaml
{{- /*
This is a comment.
*/}}
type: frobnitz
```

**Comment by FrankChen021** [comment]
Is this problem due to newer helm version?

**Comment by isandeep41** [comment]
As per the document, helm should supports both comment options, as there is no change in [template-comments-v2](https://v2.helm.sh/docs/chart_best_practices/#comments-yaml-comments-vs-template-comments) and [template-comments-v3](https://helm.sh/docs/chart_best_practices/templates/#comments-yaml-comments-vs-template-comments)


The issue got resolved after updating the license using recommended format in all templates.

**Comment by FrankChen021** [comment]
Closed via #11839
